cluster-autoscaler/cloudprovider/equinixmetal/manager_rest.go

```diff
@@ -675,13 +684,15 @@ func (mgr *equinixMetalManagerRest) deleteNodes(nodegroup string, nodes []NodeRe
 }
 
 // BuildGenericLabels builds basic labels for equinix metal nodes
-func BuildGenericLabels(nodegroup string, instanceType string) map[string]string {
+func BuildGenericLabels(nodegroup string, instanceType, instanceMetro string) map[string]string {
 	result := make(map[string]string)
 
 	result[apiv1.LabelInstanceType] = instanceType
-	//result[apiv1.LabelZoneRegion] = ""
-	//result[apiv1.LabelZoneFailureDomain] = "0"
-	//result[apiv1.LabelHostname] = ""
+	result[apiv1.LabelInstanceTypeStable] = instanceType
+	result[apiv1.LabelZoneRegion] = instanceMetro
```

Comment:
Follows the pattern set by CPEM: https://github.com/kubernetes-sigs/cloud-provider-equinix-metal/blob/036c57b7ad4585b33c5cb8dbf490d6f17cc3b05e/metal/devices.go#L78-L96